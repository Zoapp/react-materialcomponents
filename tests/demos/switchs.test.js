import React from "react";
import renderer from "react-test-renderer";
import Switchs from "@demos/components/switchs";

describe("demos/components/switchs", () => {
  it("renders correctly", () => {
    const component = renderer.create(<Switchs />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
