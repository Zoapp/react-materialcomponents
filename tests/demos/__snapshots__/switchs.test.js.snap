// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`demos/components/switchs renders correctly 1`] = `
<section>
  <h1>
    Switch examples 
  </h1>
  <div
    style={
      Object {
        "padding": "16px",
      }
    }
  >
    <div
      className="zrmc-switch-wrapper"
    >
      <div
        className="mdc-switch"
      >
        <input
          className="mdc-switch__native-control"
          defaultChecked={true}
          disabled={false}
          id="1"
          onChange={null}
          type="checkbox"
        />
        <div
          className="mdc-switch__background"
        >
          <div
            className="mdc-switch__knob"
          />
        </div>
      </div>
      <label
        className="mdc-switch-label"
        htmlFor="1"
      >
        switch 1
      </label>
    </div>
  </div>
  <div
    style={
      Object {
        "padding": "16px",
      }
    }
  >
    <div
      className="zrmc-switch-wrapper"
    >
      <div
        className="mdc-switch"
      >
        <input
          className="mdc-switch__native-control"
          defaultChecked={true}
          disabled={true}
          id="2"
          onChange={null}
          type="checkbox"
        />
        <div
          className="mdc-switch__background"
        >
          <div
            className="mdc-switch__knob"
          />
        </div>
      </div>
      <label
        className="mdc-switch-label"
        htmlFor="2"
      >
        switch 2
      </label>
    </div>
  </div>
  <div
    style={
      Object {
        "padding": "16px",
      }
    }
  >
    <div
      className="zrmc-switch-wrapper"
    >
      <div
        className="mdc-switch"
      >
        <input
          className="mdc-switch__native-control"
          defaultChecked={false}
          disabled={false}
          id="3"
          onChange={null}
          type="checkbox"
        />
        <div
          className="mdc-switch__background"
        >
          <div
            className="mdc-switch__knob"
          />
        </div>
      </div>
      <label
        className="mdc-switch-label"
        htmlFor="3"
      >
        switch 3
      </label>
    </div>
  </div>
</section>
`;
